HW0/P5/P5.txt

```diff
@@ -1 +1,41 @@
 # Your answers here.
+
+# 1. '255 seconds' -- Since they can add the first two numbers together in 1 second and then one more from the list of 254 to the cumulative total
+
+# 2. 'Yes. 34 seconds' -- Split the bags evenly between the 8 employees for 32 bags per employee, which they can add up in 31 seconds in parallel.  Then 3 seconds to merge them (8->4, 4->2, 2->1)
+
+# 3. '8 seconds.' --  2 ^ n = 256 -> n = 8 seconds, or n = log2(256)
+
+# 4. Plot Infinite Employees v. 1 Employee
+import matplotlib.pyplot as plt
+import math
+import numpy as np
+
+def y2(x):
+    new_vals = []
+    for i in x:
+        if i == 0:
+            new_vals = new_vals + [0.]
+        else:
+            new_vals = new_vals + [math.log(i, 2)] 
```

Comment:
Are you use it's just simply log?  As a thought, is it possible to have decimal seconds?